packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -446,7 +446,27 @@ class _ExpansionPanelListState extends State<ExpansionPanelList> {
         context,
         _isChildExpanded(index),
       );
-      final Row header = Row(
+      // [ExpandIcon] needs to be wrapped with MergeSemantics. If [canTapOnHeader]
```

Comment:
This comment probably needs updating to reflect that we don't use MergeSemantics in both cases? (You may also just remove the comment since it doesn't provide too much information, really)